tests/python/contrib/test_msc/test_translate_relax.py

```diff
@@ -758,6 +758,22 @@ def forward(self, data):
     _verify_model(Split(), input_info)
 
 
+def test_unbind():
```

Comment:
I'm not sure if the msc test is needed.

Reply:
I'm trying to deply a VLA model, which requires vision_backbone(DinoV2) && projector(MLP) on TensorRT and languange_model(Llama) on mlc-llm. I think MSC may be the solution, so I add test for msc